src/sinol_make/util.py

```diff
@@ -108,6 +108,43 @@ def check(path):
 		return None
 
 
+def save_config(config):
+	"""
+	Function to save nicely formated config.yml
+	"""
+
+	# The order of the fields in the config.yml file for the sake of readability
+	# The fields that are not in this list will be appended to the end of the file
```

Comment:
Two such adjacent comments make it weird when there is no dot at the end of them -- it feels like something is missing and the sentences have a broken flow. What do you think about adding the dots and also adding dots in the other comments in the files to make it consistent?

Reply:
I added dots to every comment in this function 24b28f1f13e26bc11b87306f79a0f31fd8f736a6